src/job_handler_plugins/local_container/__init__.py

```diff
@@ -64,9 +64,11 @@ def start(self) -> str:
     def remove(self) -> Tuple[str, str]:
         try:
             container = self.client.containers.get(self.local_container_name)
+            if container.status != "exited":
+                container.kill()
             container.remove()
-        except docker.errors.NotFound:
-            pass
+        except Exception as err:
+            self.job.status, str(err)
```

Comment:
Should not catch "Exception" inside handlers.  We have no idea what happens then, for example, the container might not be removed.